### ISSUE TITLE
Installation on Fedora requires extra setup steps

### ISSUE THREAD

**Issue by geerlingguy** [geerlingguy/drupal-vm#301]
While, technically, this step is related to VirtualBox installation and not Drupal VM, I think it may be good to start a `linux.md` section in the docs and start putting distro-specific notes and installation gotchas into there (Ubuntu, Fedora, and Arch are the three OSes I see the most small issues for).

The first thing I've noticed while working within Fedora 22 (I'll upgrade to 23 in a day or two) is that to resolve the following error on `vagrant [anything]`:

```
$ vagrant status
VirtualBox is complaining that the kernel module is not loaded. Please
run `VBoxManage --version` or open the VirtualBox GUI to see the error
message which should contain instructions on how to fix this error.
```

...I had to do the following:

```
sudo dnf install "kernel-devel-uname-r == $(uname -r)"
sudo dnf install gcc
sudo /var/lib/vboxdrv.sh setup
```
Basically, some of the VirtualBox deps were missing, including kernel headers, and I had to install them prior to VirtualBox actually running.

**Comment by kostajh** [comment]
@geerlingguy I've noted that whenever I run a kernel upgrade, I need to run `sudo dnf remove VirtualBox` followed by `sudo dnf install VirtualBox`.
